test/sai_test/sai_lag_test.py

```diff
@@ -60,10 +60,10 @@ def load_balance_on_src_ip(self):
                                      ip_src=self.servers[2][1].ipv4,
                                      ip_id=105,
                                      ip_ttl=63)
-        send_packet(self, 1, pkt1)
+        send_packet(self, self.dut.port_obj_list[1].dev_port_index, pkt1)
         verify_packet_any_port(
             self, exp_pkt1, self.servers[11][1].l3_lag_obj.member_port_indexs)
-        send_packet(self, 1, pkt2)
+        send_packet(self, self.dut.port_obj_list[1].dev_port_index, pkt2)
         verify_packet_any_port(
             self, exp_pkt2, self.servers[11][1].l3_lag_obj.member_port_indexs)
```

Comment:
all the port index need to convert to dev_port_index. 
there is a new method get_dev_port_indexes, which can convert the port indexes to dev_port_indexes

Reply:
Already change this in newest commit.